data/human/hails.txt

```diff
@@ -1708,6 +1710,77 @@ phrase "friendly civilian"
 		", but I still had to take a big loan to buy my first ship."
 		", but it was my grandma's inheritance that allowed me to buy this ship."
 
+phrase "friendly civilian"
```

Comment:
Same. I think most of these combinations are kind of silly and mostly nonsensical, sorry.

Reply:
I removed all but 4 of them. Let me know if the last 4 also need to go.

Reply:
The remaining ones seem ok to me. :)